pkg/gadgets/trace/open/tracer/bpf/opensnoop.bpf.c

```diff
@@ -121,6 +123,12 @@ int trace_exit(struct trace_event_raw_sys_exit* ctx)
 	event.mntns_id = gadget_get_mntns_id();
 	event.timestamp = bpf_ktime_get_boot_ns();
 
+	// Attempting to extract the full file path with symlink resolution
+	if (0 <= ret)
+	{
+		read_full_path_of_open_file_fd(ret, (char*)event.fname, sizeof(event.fname));
```

Comment:
I agree with @eiffel-fl, we should use a different field for the given path and the resolved path.

pkg/gadgets/trace/open/types/types.go could have both `Path` and `FullPath`. One of them could have the column attribute "hide" if it is too long to print on one line.

Reply:
@alban , see my comments below. Don't you think it will create pressure on the ring buffer and event structures? This would effectively double the size of an event record. Besides I am not sure about how useful the `Path` would be in this case.